client/src/templates/client-side/statsTable.html

```diff
@@ -0,0 +1,25 @@
+{# Not .njk because https://github.com/mozilla/nunjucks/pull/1225 #}
+{% for row in table %}
+  {% if loop.first %}
+    <tr>
+      {% for content in row %}
+        <th onclick="handleSort(this)">
+          {{ content }}
+          {% if currentSortingColumn == loop.index-1 %}
+            {% if isAscending %}
+              ⬆
+            {% else %}
+              ⬇️
+            {% endif %}
```

Comment:
Add an up arrow if we are sorting this column ascending, or a down arrow if descending.